frontend/projects/upgrade/src/styles.scss

```diff
@@ -249,6 +249,10 @@ button[mat-icon-button] {
   width: 805px;
 }
 
+.full-width {
+  width: 100%;
+}
+
```

Comment:
Could we remove the `full-width` class for now until we determine it's widely used and needed? I think it might be better to add this later.

Reply:
i added it precisely because it was becoming a common pattern, search for `class="full-width"` in the feature-flags project files.

that being said, I'm kind of an idiot when it comes to organizing css styles, so I'm all ears for better ideas. I can localize it to the component if that's what you'd prefer for now.